frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/details-card.tsx

```diff
@@ -53,6 +62,21 @@ const DetailsCard: React.FC<DashboardItemProps> = ({
     InfrastructureModel,
     'cluster',
   );
+  const [ocsSubscription, ocsSubscriptionLoaded, ocsSubscriptionError] = useK8sGet<
```

Comment:
```suggestion
  const [ocsSubscription, ocsSubscriptionLoaded, ocsSubscriptionError] = useK8sWatch<
```
We want this to be updated. Please make changes for all instances.

Reply:
Yeah! Right! We should watch